src/test/java/com/iexec/common/docker/DockerRunRequestTests.java

```diff
@@ -75,17 +76,17 @@ void shouldAddDeviceThenAddSgxDevice() {
         DockerRunRequest request = DockerRunRequest.builder()
                 .containerName("containerName")
                 .devices(devices) // changed order here
-                .isSgx(true)
+                .sgxDriverMode(SgxDriverMode.LEGACY)
                 .build();
         System.out.println(request);
```

Comment:
Should we remove this or replace it with a SLF4J logger call ?

Reply:
Replaced with Slf4j in https://github.com/iExecBlockchainComputing/iexec-common/pull/308/commits/24b0c04479f5ce6a5963a0c09882a81f478e7c41.